### PR TITLE
Composer Package Added

### PR DESCRIPTION
**Pull request by imjalpreet**
@oparoz Have a look at the composer.json file. I have made some of the changes that you had asked for.

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  "require-dev": {
+    "codeception/codeception": "2.1.*",
+    "codeception/phpbuiltinserver": "*",
+    "codeception/c3": "2.*",
+    "flow/jsonpath": "^0.3.1",
+    "phpdocumentor/phpdocumentor": "^2.8",
+    "jakub-onderka/php-parallel-lint": "0.*",
+    "jakub-onderka/php-console-highlighter": "0.*"
+  }
+}
```
Comment (oparoz) [inline composer.json]: Please add an extra empty line at the end

**Comment by oparoz** [comment]
Have you tested it?

**Comment by imjalpreet** [comment]
@oparoz I have tested it and its working.

**Comment by oparoz** [comment]
Confirmed.

**Pull request merged**